server/webhook.go

```diff
@@ -83,7 +83,6 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, body
 
 	post, appErr := p.API.GetPost(postID)
 	if appErr != nil {
-		p.API.LogWarn("Could not get meeting post by id", "err", appErr)
```

Comment:
This could be a result of the `AppError` struct having weird non-nil behavior. If we use `p.client` instead throughout the function, that would ensure no issues in that regard

Reply:
@mickmister I found the issue, we were returning `appErr` on [line 153](https://github.com/mattermost/mattermost-plugin-zoom/blob/d73eafada987e2458d721f809ee18d750551ef3e/server/store.go#L153) which was "nil". Updated the code to return an actual error at that point. Also updated it to use `p.client`.